packages/cannon-worker-api/README.md

```diff
@@ -0,0 +1,5 @@
+# @pmndrs/cannon-worker-api
+
+Web worker api for [cannon-es](https://github.com/pmndrs/cannon-es). Currently used in `@react-three/fiber`.
+
+**This package is not intended for direct use** - see `@react-three/cannon`
```

Comment:
I think we are intending it to be used, no?

Reply:
My current thinking is that we'll end up with these packages:

**@react-three/cannon**
Existing package
**@pmndrs/cannon-worker**
User-facing package for using cannon-es running in a web worker.



@pmndrs/cannon-worker-api

Reply:
I guess what I was trying to say is it's likely not the package people are looking for - it's a shared core package.

I think our end goal is having these packages:

**@pmndrs/cannon-worker-api**
Contains some types, [`web-worker-api.ts`](https://github.com/pmndrs/use-cannon/blob/5ff2baacd7caff2079b21b42ea9ca7798070a5d9/src/cannon-worker-api.ts) and the [web worker worker code](https://github.com/pmndrs/use-cannon/tree/5ff2baacd7caff2079b21b42ea9ca7798070a5d9/src/worker)

**@react-three/cannon**
Existing package, uses the web worker API in `@pmndrs/cannon-worker-api`

**@pmndrs/cannon-worker**
New user-facing package for use with three.js directly, with similar usage to vanilla cannon-es.
Will also use `@pmndrs/cannon-worker-api`.

Unless someone is looking to make their own cannon web worker package for some other framework, they likely wouldn't use `@pmndrs/cannon-worker-api` directly?

Reply:
Discussed offline:

The cannon-es like API will gradually be built out in `@pmndrs/cannon-worker-api`.

There won't be another separate `@pmndrs/cannon-worker` package.

Current path forward:

- Move cannon-worker-api.ts / CannonWorkerApi and web worker into @pmndrs/cannon-worker-api
- Build out @pmndrs/cannon-worker-api cannon-es like api
- Incrementally replace direct usage of CannonWorkerApi with the new cannon-es like api

Planning on a v5 release that will include the worker API as a separate package.